rpc/eth_test.go

```diff
@@ -107,14 +110,34 @@ func TestWeb3Service(t *testing.T) {
 	t.Run("EthTransactions", func(t *testing.T) {
 		var txHash, contractAddress string
 
+		sender := genesisAccounts[1]
 		receivee := genesisAccounts[2].GetPublicKey().GetAddress()
 		acc, err := kern.State.GetAccount(receivee)
 		require.NoError(t, err)
 		before := acc.GetBalance()
 
 		t.Run("EthSendRawTransaction", func(t *testing.T) {
-			// see: https://github.com/ethereumjs/ethereumjs-tx/blob/master/examples/transactions.ts#L9
-			raw := `0xf867808082520894f97798df751deb4b6e39d4cf998ee7cd4dcb9acc880de0b6b3a76400008025a0f0d2396973296cd6a71141c974d4a851f5eae8f08a8fba2dc36a0fef9bd6440ca0171995aa750d3f9f8e4d0eac93ff67634274f3c5acf422723f49ff09a6885422`
```

Comment:
This was the main thing I needed to fix since our secp generated addresses had changed I didn't know how to recreate this